2-python-fundamentals-i.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `2-python-fundamentals-i.ipynb` Cell 3 Line 3_
```python
my_var = 42
print(my_var)
print(type(my_var));
```
Ditch the `;`